iterativeTrim.py

```diff
@@ -195,6 +197,26 @@ def writeChConfig(chConfig,vfat,vfatID,trimDAC,trimPol,mask,maskReason):
     vfatBoard = HwVFAT(cardName, args.link, args.debug)
     print("Opened connection")
 
+    if args.vfatConfig is not None:
+        try:
+            print('Configuring VFAT Registers based on {0}'.format(args.vfatConfig))
+            vfatTree = r.TTree('vfatTree','Tree holding VFAT Configuration Parameters')
+            vfatTree.ReadFile(args.vfatConfig)
+            
+            for event in vfatTree :
+                # Skip masked vfats
+                if (args.vfatmask >> int(event.vfatN)) & 0x1:
+                    continue
+                    
+                # Write CFG_THR_ARM_DAC
+                print('Set link {0} VFAT{1} CFG_THR_ARM_DAC to {2}'.format(args.link,event.vfatN,event.vt1))
+                vfatBoard.setVFATThreshold(chip=int(event.vfatN), vt1=int(event.vt1))
+        except IOError as e:
+            print('{0} does not seem to exist or is not readable'.format(options.filename))
```

Comment:
One thing I didn't ask before, if this exception is raised, what is the resulting behaviour (it doesn't look like an exit or something like that)?

Reply:
First, `options.filename` should be `options.vfatConfig`. 

The resulting behavior if the exception is raised is that it will whatever thresholds the VFATs happen to be configured with when it is run.